openpype/hosts/maya/plugins/publish/extract_multiverse_usd.py

```diff
@@ -156,6 +156,14 @@ def filter_members(self, members):
         return members
 
     def process(self, instance):
+        # check if there is "usd" in families because of
+        # `ExtractMultiverseUsdAnim` that inherits this and should
+        # run on animation family too.
+        families = set(instance.data["families"])
+        families.add(instance.data["family"])
+        if "usd" not in families:
+            return
+
```

Comment:
Should this not be happening anyways with the class attribute `families`?

Reply:
Problem is actually here:
https://github.com/ynput/OpenPype/blob/cce7c3be0839ff9eb2b603dd6239ce777ddaeec6/openpype/hosts/maya/plugins/publish/extract_multiverse_usd.py#L257

The second class defined there is using the same `process()` but defines class attribute families as just `["animation"]`

Reply:
Ah that makes more sense.

Reply:
What about setting the other's class families like:
```
families = ["usd", "animation"]
```

And customizing its matching algorithm:
```python
matching = pyblish.api.Exact
```

Or even:
```python
matching = pyblish.api.Subset
```

---

Or if we're just trying to resolve for the other plug-in's process, how about actually doing that?

```python
...
    def process(self, instance):
        families = set(instance.data["families"])
        families.add(instance.data["family"])
        if "usd" not in families:
            return
    
        super(ExtractMultiverseUsdAnim, self).process(instance)
...
```

However, I'd recommend the solution mentioned above using another mathing algorithm.

Reply:
Cool, I've actually never used these, good to know it is there!